react/AppSections/components/AppsSection.jsx

```diff
@@ -18,23 +25,38 @@ export const AppsSection = ({
 }) => {
   const { isMobile } = useBreakpoints()
   const { t } = useI18n()
+  const apps = useMemo(() => makeAppsListNameLowerCased(appsList), [appsList])
+  const getAppBySlug = useMemo(
+    () => slug => appsList.find(app => app.slug === slug),
+    [appsList]
+  )
+
   return (
     <div className={styles.AppsSection}>
       {subtitle}
-      {appsList && !!appsList.length && (
+      {apps && !!apps.length && (
         <div className={styles.AppsSection__list}>
-          {sortBy(appsList, makeNameGetter(t)).map(app => (
-            <AppTile
-              app={app}
-              namePrefix={getTranslatedManifestProperty(app, 'name_prefix', t)}
-              name={getTranslatedManifestProperty(app, 'name', t)}
-              onClick={() => onAppClick(app.slug)}
-              key={app.slug}
-              showDeveloper={!isMobile}
-              displaySpecificMaintenanceStyle={displaySpecificMaintenanceStyle}
-              IconComponent={IconComponent}
-            />
-          ))}
+          {sortBy(apps, makeNameGetter(t)).map(app => {
```

Comment:
why not moving the sortBy inside useMemo? that operation (sorting) costs and chould benefit of memo, here.

Reply:
thanks. Done